src/components/NearMe/NearMeProfiles.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect, useState } from 'react'
+import { Box, Grid, ImageList, ImageListItem, Typography } from '@mui/material'
+import { LocationOn } from '@mui/icons-material'
+import { makeStyles } from 'tss-react/mui'
+import { getUsersNearMeData } from '../../actions/userServices'
+import { picProfileColMob, picProfileColWeb } from './enums'
+
+type isMobileProps = {
+  isMobile: boolean
+}
+type userNearMeObjectType = {
+  id: string
+  location: string
+  picture: string
+  fName: string
+  lName: string
+}
+const NearMeProfiles = ({ isMobile }: isMobileProps) => {
+  const { classes } = useStyles()
+  const [list, setList] = useState<Array<userNearMeObjectType>>([])
+  const columns = isMobile ? picProfileColMob : picProfileColWeb
+  const txtAlign = isMobile ? 'center' : 'left'
+
+  useEffect(() => {
```

Comment:
e.g. how to do side calls in the hook
`  useEffect(() => {
    async function fetchData() {
      const result = await userServices()
      setSuccess(result)
    }
      fetchData()
  })`

Reply:
Refactored data fetching from a file following a provided example.